### ISSUE TITLE
CTC Loss参数的疑问？

### ISSUE THREAD

**Issue by wstchh** [PaddlePaddle/PaddleOCR#11743]
![`39_I7(9R7F%~{X4HTSC9LR](https://github.com/PaddlePaddle/PaddleOCR/assets/67768819/7699e68e-76ef-4f30-8af1-5edecf8a4967)

# loss = ctc_loss(log_probs, targets, input_lengths, target_lengths)
log_probs一般需要经过torch.nn.functional.log_softmax处理后再送入到CTCLoss中；

网上说的必须要经过log_softmax,  但是貌似paddleocr中是直接输出的，没有经过log_softmax

**Comment by changdazhou** [comment]
paddle的ctc_loss把softmax写进去了，可以参考一下这个https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/CTCLoss_cn.html#ctcloss

**Comment by nissansz** [comment]
rec resnet34可以用哪个loss改善识别结果准确率？
rec_cppd_loss.py
rec_ctc_loss.py
rec_enhanced_ctc_loss.py
rec_multi_loss.py
rec_nrtr_loss.py
